substrate/frame/revive/src/lib.rs

```diff
@@ -1799,6 +1829,13 @@ impl<T: Config> Pallet<T> {
 		Some(T::AddressMapper::to_address(&account_id))
 	}
 
+	#[cfg(feature = "runtime-benchmarks")]
```

Comment:
let's just update the function to return an H160 and default_or_unwrap
we will keep the block_author function in runtime_api as it is for now to not create any breaking change in the substrate rpc APIS